pkg/operator/ceph/object/controller.go

```diff
@@ -159,6 +169,11 @@ func (c *ObjectStoreController) createOrUpdateStore(objectstore *cephv1.CephObje
 }
 
 func (c *ObjectStoreController) onDelete(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Deleting object store for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.